index.js

```diff
@@ -1,8 +1,15 @@
 'use strict';
 var isRoot = require('is-root');
+var DEFAULT_UID = {
+	darwin: 501,
+	freebsd: 1000,
+	linux: 1000,
+	sunos: 100
+};
```

Comment:
What should it do when there's a platform not in this list?

We should probably:

```js
if (isRoot() && process.setuid && process.platform in DEFAULT_UID) {
```